<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/ic_green.png</normaloff>images/ic_green.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Configurazione</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>491</width>
       <height>380</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>380</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>17</height>
      </size>
     </property>
     <widget class="QWidget" name="tab1">
      <property name="accessibleName">
       <string/>
      </property>
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/cbreak.png</normaloff>:/images/cbreak.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Moduli e Porte</string>
      </attribute>
      <widget class="QTreeView" name="treeView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>471</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="icon">
       <iconset>
        <normaloff>images/lamp.png</normaloff>images/lamp.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Porte attive</string>
      </attribute>
      <widget class="QListView" name="ruleList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="icon">
       <iconset>
        <normaloff>images/runner.png</normaloff>images/runner.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Regole</string>
      </attribute>
      <widget class="QTreeView" name="View">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>451</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Dispositivo</string>
    </property>
    <widget class="QLabel" name="lb_hardware">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Firmware.........:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_cycle">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Memoria..........:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ciclo (ms)........:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hardware.........: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_memory">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_firmware">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial.............:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_serial">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stato MyPlc.....:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_status">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Progetto</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_filename">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_datetime">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifica:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_title">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Titolo:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Configurazione</string>
    </property>
    <widget class="QLabel" name="lb_modules">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Porte OUT.......:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Moduli ............:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_pin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_rules">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Porte IN..........:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Regole...........:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_pout">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Aiuto</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Strumenti</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionFlash_Nanoboard"/>
   </widget>
   <widget class="QMenu" name="menuMyPlc">
    <property name="title">
     <string>MyPlc</string>
    </property>
    <addaction name="actionAddModule"/>
    <addaction name="separator"/>
    <addaction name="actionSerialConnect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMyPlc"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileOpen"/>
   <addaction name="actionFileSave"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionSerialConnect"/>
   <addaction name="separator"/>
   <addaction name="actionAddModule"/>
   <addaction name="separator"/>
   <addaction name="actionClean"/>
  </widget>
  <action name="actionAddModule">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/addmodule.png</normaloff>:/images/addmodule.png</iconset>
   </property>
   <property name="text">
    <string>Aggiungi Nuovo Modulo</string>
   </property>
   <property name="toolTip">
    <string>Aggiungi Modulo </string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Esci</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/f_open2.png</normaloff>
     <normalon>images/open.png</normalon>:/images/f_open2.png</iconset>
   </property>
   <property name="text">
    <string>Apri...</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/f_save.png</normaloff>
     <normalon>images/save (2).png</normalon>:/images/f_save.png</iconset>
   </property>
   <property name="text">
    <string>Salva</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/f_save_as.png</normaloff>
     <normalon>images/save_as (2).png</normalon>:/images/f_save_as.png</iconset>
   </property>
   <property name="text">
    <string>Salva come...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Informazioni</string>
   </property>
  </action>
  <action name="actionSerialConnect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Collega</string>
   </property>
   <property name="toolTip">
    <string>Connessione a unità MyPlc</string>
   </property>
  </action>
  <action name="actionFlash_Nanoboard">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ic_flash.png</normaloff>:/images/ic_flash.png</iconset>
   </property>
   <property name="text">
    <string>Flash Nanoboard</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>actionClean</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
